companion/lib/Service/HttpApi.ts

```diff
@@ -320,6 +320,14 @@ export class ServiceHttpApi {
 		// surfaces
 		this.#apiRouter.post('/surfaces/rescan', this.#surfacesRescan)
```

Comment:
Adding reporting of surface state (e.g. offline) would also be really nice, but maybe that's a separate issue/PR?